### PR TITLE
Implement App Icon

### PR DESCRIPTION
**Pull request by esthcarelle**
### Dialer App icon

Screenshot from 2024-02-13 16-50-28
image

PS: @cedricbahirwe The dimensions are not matching google's dimens 
FYI: https://developer.android.com/distribute/google-play/resources/icon-design-specifications

### CODE REVIEW

**Comment by esthcarelle** [comment]
@cedricbahirwe I am not aware of any that does that, but maybe resizing it to the mentioned dimensions with any available tool would do
Example of one that works: https://hotpot.ai/icon-resizer

**Comment by esthcarelle** [comment]
@cedricbahirwe 
Screenshot from 2024-02-18 20-28-45

**Comment by esthcarelle** [comment]
@cedricbahirwe Can you review?

**Comment by cedricbahirwe** [comment]
> @cedricbahirwe Can you review?

On it

File: TheDialer/.idea/gradle.xml [hunk 1]
```diff
@@ -4,15 +4,15 @@
   <component name="GradleSettings">
     <option name="linkedExternalProjectsSettings">
       <GradleProjectSettings>
-        <option name="testRunner" value="GRADLE" />
-        <option name="distributionType" value="DEFAULT_WRAPPED" />
         <option name="externalProjectPath" value="$PROJECT_DIR$" />
+        <option name="gradleJvm" value="#GRADLE_LOCAL_JAVA_HOME" />
```
Comment (cedricbahirwe) [inline TheDialer/.idea/gradle.xml:8]: Don't know what this is doing, but hope it won't cause issue

**Review by cedricbahirwe — APPROVED** [review]

File: TheDialer/.idea/gradle.xml (hunk 1 above)
Comment (esthcarelle) [reply TheDialer/.idea/gradle.xml:8]: it is fine, it won't affect your environment,it's my gradle file

**Pull request merged**